gensim/models/ldamulticore.py

```diff
@@ -289,7 +289,7 @@ def process_result_queue(force=False):
                 chunk_put = False
                 while not chunk_put:
                     try:
-                        job_queue.put((chunk_no, chunk, self), block=False, timeout=0.1)
+                        job_queue.put((chunk_no, chunk, self.state), block=True, timeout=0.1)
```

Comment:
why `block=True`?

Reply:
because in this place we should or remove timeout or set `block` to `True`. As said in [docs](https://docs.python.org/2/library/multiprocessing.html#multiprocessing.Queue.put) at `Block=False` timeout is ignored. In case when queue is full this `put` can be repeated very often and eat CPU, so I just make it blocked.